manifests/params.pp

```diff
@@ -102,6 +104,8 @@
       $init_config_tmpl = 'foreman.sysconfig'
       $puppet_etcdir = '/etc/puppet'
       $puppet_home = '/var/lib/puppet'
+      $jobs_service = $dynflow_in_core ? { true => 'dynflow-executor',
```

Comment:
Since here ```$dynflow_in_core``` is always true, you can just write ```$jobs_service = 'dynflow-executor```. If you want to dynamically determine this it should be in the ```foreman::service::jobs``` class. I'd recommend ```$jobs_service = undef``` here and then determine it later.